content/background.html

```diff
@@ -87,7 +87,7 @@ <h3> International Students </h3>
   International students came from countries such as China and India.
 </p>
 <div class="half second">
-  <span class="graph-title">Proportion of international students</span>
+  <span class="graph-title">nationality breakdown</span>
```

Comment:
I don't think we should use the word nationality here since the graph shows only the number of international students and domestic students. And students could have multiple nationalities

Reply:
changed to "domestic (Canadian) vs international student breakdown" in e30f631. the reason i had changed it was because the graph displays "Canadian" and "international", which imo didn't quite match the graph title..? you're right tho, in that we only see whether a student was canadian or not (open to other suggestions of graph titles)